esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -151,7 +195,7 @@ IPropertyTree * CWsESDLConfigEx::getEspProcessRegistry(const char * espprocname,
     if (!globalLock || !globalLock->queryRoot())
         throw MakeStringException(-1, "Unable to connect to ESP configuration information in dali %s", xpath.str());
 
-    globalLock->close(false);
+    globalLock->close();
```

Comment:
globalLock.clear() would be better

Reply:
The getEspProcessRegistry() method creates unnecessary overhead, I'll remove the method.